lib/translator/common.h

```diff
@@ -16,6 +16,7 @@
 #define LIB_TRANSLATOR_COMMON_H
 
 #include <cstring>
+#include <new>
```

Comment:
I think this can just be placed in unmap.cc

Reply:
It's required here because it's used by SafePointerCastWrite. Should have been in the original PR but for some reason tests didn't catch it